xml/admin_ceph_upgrade.xml

```diff
@@ -1397,6 +1397,147 @@ data:
+       <para>If one OSD fails to convert, re-running the rebuild destroys the
+         already converted &bluestore; OSDs. If this happens, run:</para>
+         <screen>
+           &prompt.smaster; salt-run disks.deploy <replaceable>TARGET</replaceable>
+         </screen>
+       </important>
+ </tip>
+  <procedure>
+   <step>
+    <para>
+     Migrate hardware profiles:
+    </para>
+ <screen>&prompt.smaster;salt-run state.orch ceph.migrate.policy</screen>
+    <para>
+     This runner migrates any hardware profiles currently in use by the
+     <filename>policy.cfg</filename> file. It processes
+     <filename>policy.cfg</filename>, finds any hardware profile using the
+     original data structure, and converts it to the new data structure. The
+     result is a new hardware profile named
+     'migrated-<replaceable>ORIGINAL_NAME</replaceable>'.
+     <filename>policy.cfg</filename> is updated as well.
+    </para>
+    <para>
+     If the original configuration had separate journals, the &bluestore;
+     configuration will use the same device for the 'wal' and 'db' for that
+     OSD.
+    </para>
+   </step>
+   <step>
+    <para>
+     &deepsea; migrates OSDs by setting their weight to 0 which 'vacuums' the
+     data until the OSD is empty. You can either migrate OSDs one by one, or
+     all OSDs at once. In either case, when the OSD is empty, the
+     orchestration removes it and then re-creates it with the new
+     configuration.
+    </para>
+    <tip>
+     <title>Recommended Method</title>
+     <para>
+      Use <command>ceph.migrate.nodes</command> if you have a large number of
+      physical storage nodes or almost no data. If one node represents less
+      than 10% of your capacity, then the
+      <command>ceph.migrate.nodes</command> may be marginally faster moving
+      all the data from those OSDs in parallel.
+     </para>
+     <para>
+      If you are not sure about which method to use, or the site has few
+      storage nodes (for example each node has more than 10% of the cluster
+      data), then select <command>ceph.migrate.osds</command>.
+     </para>
+    </tip>
+    <substeps>
+     <step>
+      <para>
+       To migrate OSDs one at a time, run:
+      </para>
+ <screen>&prompt.smaster;salt-run state.orch ceph.migrate.osds</screen>
+     </step>
+     <step>
+      <para>
+       To migrate all OSDs on each node in parallel, run:
+      </para>
+ <screen>&prompt.smaster;salt-run state.orch ceph.migrate.nodes</screen>
+     </step>
+    </substeps>
+    <tip>
+     <para>
+      As the orchestration gives no feedback about the migration progress, use
+     </para>
+ <screen>&prompt.cephuser;ceph osd tree</screen>
+     <para>
+      to see which OSDs have a weight of zero periodically.
+     </para>
+    </tip>
+   </step>
+  </procedure>
+  <para>
+   After the migration to &bluestore;, the object count will remain the same
+   and disk usage will be nearly the same.
+  </para>
+</sect1>
+
```

Comment:
Likewise, the states `ceph.migrate.nodes` and `ceph.migrate.osds` exist, but the `osd.redeploy` module is absent in SES6.

These paragraphs can be removed as well.